functions/src/utils/parsers.ts

```diff
@@ -1,19 +1,14 @@
-import { format } from "date-fns";
-
-export const EMPTY_VALUE = "-";
-
-export function createUpdateLink(token: string): string {
-  return `http://purebhakti.info/audioseva/form/sound-quality-report?token=${token}`;
-}
+// import { format } from "date-fns";
```

Comment:
Guess what?

Reply:
Meant to backspace, not sure how it got commented out.